app/views/refinery/authentication/devise/admin/users/_actions.html.erb

```diff
@@ -1,7 +1,5 @@
-<ul>
-  <li><%= render '/refinery/admin/search', :url => refinery.authentication_devise_admin_users_path -%></li>
```

Comment:
Sure, I can just put the old one back in temporarily until the new one replaces it.  One minute here.